src/main.ts

```diff
@@ -124,6 +220,48 @@ export async function run(args: ArgsType) {
   );
 
   switch (args.command) {
+    case 'shipit': {
+      const version = await getVersion(githubRelease, args);
+
+      if (version === '') {
```

Comment:
shouldn't the `version` be defined as a semver bump?
```suggestion
      if (version !== '') {
```